File: C++/1018/1018.cpp
#include <cstdio>
#include <iostream>
#include <type_traits>
using namespace std;

int main() {
	int x, y;
	double value;

	cin >> x;

	cout << x << endl;

	value = x / 100;
	y = x % 100;
	printf("%.0f nota(s) de R$ 100,00\n", value);
	
	value = y / 50;
	y = y % 50;
	printf("%.0f nota(s) de R$ 50,00\n", value);
	
	value = y / 20;
	y = y % 20;
	printf("%.0f nota(s) de R$ 20,00\n", value);
	
	value = y / 10;
	y = y % 10;
	printf("%.0f nota(s) de R$ 10,00\n", value);

	value = y / 5;
	y = y % 5;
	printf("%.0f nota(s) de R$ 5,00\n", value);
	
	value = y / 2;
	y = y % 2;
	printf("%.0f nota(s) de R$ 2,00\n", value);
	
	value = y / 1;
	y = y % 1;
	printf("%.0f nota(s) de R$ 1,00\n", value);
}

File: Python/1018.py
# 100, 50, 20, 10, 5, 2, 1

nota = int(input())

print(nota)
print("%d nota(s) de R$ 100,00" % (nota/100))
nota %= 100
print("%d nota(s) de R$ 50,00" % (nota/50))
nota %= 50
print("%d nota(s) de R$ 20,00" % (nota/20))
nota %= 20
print("%d nota(s) de R$ 10,00" % (nota/10))
nota %= 10
print("%d nota(s) de R$ 5,00" % (nota/5))
nota %= 5
print("%d nota(s) de R$ 2,00" % (nota/2))
nota %= 2
print("%d nota(s) de R$ 1,00" % (nota/1))
nota %= 1
